builtin/logical/transit/path_random.go

```diff
@@ -45,6 +45,7 @@ func (b *backend) pathRandom() *framework.Path {
 
 func (b *backend) pathRandomWrite(ctx context.Context, req *logical.Request, d *framework.FieldData) (*logical.Response, error) {
 	bytes := 0
+	const maxBytes = 128 * 1024
```

Comment:
Please put this at the package level so that you can refer to it from both the test and here.

Reply:
Solved in https://github.com/hashicorp/vault/pull/7144/commits/2699e4ea2abc9282ac263687148902a704d6bc4e